neurom/exceptions.py

```diff
@@ -29,29 +29,14 @@
 """Module containing NeuroM specific exceptions."""
 
 
-class NeuroMError(Exception):
-    """Base class for all NeuroM exceptions."""
+# pylint: disable=unused-import, import-error
```

Comment:
This is a topic to discuss. Do we want to hide MorphIO  usage from NeuroM users?

Reply:
i think if morphio crashes on loading/writing, the user should see a morphio error, so it knows it is coming from morphio, no?

Reply:
I don't know. For some reason Benoit decided to map all MorphIO exceptions as NeuroM exceptions.

Reply:
Not sure what is best for debugging, etc... I don't have a strong opinion on that.